201-front-door-create-caching/azuredeploy.json

```diff
@@ -85,9 +88,12 @@
                             "patternsToMatch": [
                                 "/site2/*"
                             ],
-                            "forwardingProtocol": "MatchRequest",
-                            "backendPool": {
-                                "id": "[concat(variables('frontdoorref'),'/backendPools/',parameters('backendPools2').name)]"
+                            "routeConfiguration": {
+                                "@odata.type": "#Microsoft.Azure.FrontDoor.Models.FrontdoorForwardingConfiguration",
+                                "forwardingProtocol": "MatchRequest",
+                                "backendPool": {
+                                    "id": "[concat(variables('frontdoorref'),'/backendPools/',parameters('backendPools2').name)]"
```

Comment:
remove concat and use resourceId

Reply:
Done, I've updated the PR to use resourceId instead of concat:

"backendPool": {
                                    "id": "[resourceId('Microsoft.Network/frontDoors/backendPools', parameters('frontDoorName'), parameters('backendPools2').name)]"
                                }